src/components/Stepper/index.tsx

```diff
@@ -30,9 +32,10 @@ export default function Stepper({
   disableInput,
   size,
   theme,
-  loading
+  loading,
+  onChange
 }: IProps) {
-  const [value, setValue] = useState(0);
+  let [value, setValue] = useState(0);
```

Comment:
why using let?

Reply:
Because I need to pass the value as parameter and I need value+ and value-.